libs/ui/src/ui_strings/app_strings.tsx

```diff
@@ -639,15 +639,21 @@ export const appStrings = {
     <UiString uiStringKey="labelBallotStyle">Ballot Style:</UiString>
   ),
 
-  labelBmdPatCalibrationInputIdentified: () => (
-    <UiString uiStringKey="labelBmdPatCalibrationInputIdentified">
-      Input Identified:
+  labelBmdPatCalibrationChooseInput: () => (
+    <UiString uiStringKey="labelBmdPatCalibrationChooseInput">
+      Choose Input:
```

Comment:
"Choose" feels a bit misleading because they don't get to actually pick which input corresponds to which action right?

Reply:
They actually do - dual switch devices can be set up either way. You can sip to select or sip to move, and vice versa. That's part of the purpose of the calibration / identification process. We thought that "choose" felt more accurate than "identify" for that reason. cc @mattroe.

Reply:
Ah ok, in that case sounds good

Reply:
Oh, I thought it prompted you to active a specific input (e.g. Move) and then if you activated the other input it showed you the warning with the exclamation point icon saying that you activated the wrong one

Reply:
@kshen0 can you help clarify this?

Reply:
When you specify an input, you have to make the input twice. I think the goal here is just to confirm and make sure the wrong input is not selected. You get the "wrong input" screen if your confirmation input is different than your selection input.

Reply:
Ah ok thanks for clarifying